pkg/avx/collector.go

```diff
@@ -38,6 +40,8 @@ var (
 			"cgroup",
 		}, nil,
 	)
+
+	BpfBinaryName = "avx512.o"
```

Comment:
for now this can be `const`.

Reply:
AFAIK `const`s cannot be overridden using `-ldflags -X` which was my idea to keep available..

Reply:
I see. Let's have it var then